chef/cookbooks/apache2/templates/suse/mods/ssl.conf.erb

```diff
@@ -38,17 +56,15 @@
 	#   Note that on most platforms shared memory segments are not allowed to be on 
 	#   network-mounted drives, so in that case you need to use the dbm method.
 	#SSLSessionCache        none
+
+	#<IfModule socache_dbm>
 	#SSLSessionCache         dbm:/var/lib/apache2/ssl_scache
-	#SSLSessionCache        shmht:/var/lib/apache2/ssl_scache(512000)
-	SSLSessionCache         shmcb:/var/lib/apache2/ssl_scache(512000)
-	SSLSessionCacheTimeout  600
+	#</IfModule>
 
-	#   This configures the SSL engine's semaphore (aka. lock) which is
-	#   used for mutual exclusion of operations which have to be done in a
-	#   synchronized way between the pre-forked Apache server processes.
-	#   "default" tells the SSL Module to pick the default locking
-	#   implementation as determined by the platform and APR.
-	SSLMutex  default
```

Comment:
Dunno.  If we do, it needs to go inside <IfVersion>, which means an `a2enmod version` somewhere.